app/controllers/api/v1/animation_datas_controller.rb

```diff
@@ -3,18 +3,15 @@ module V1
     class AnimationDatasController < ApplicationController
       skip_before_action :authenticate_user!
       def index
-        
-        @animation_record = UserAnimation.find_by("user_id = ? AND location = ? AND start_date <= ? 
+        @animation_record = UserAnimation.find_by("user_id = ? AND location = ? AND start_date <= ?
```

Comment:
you don't need to write query like that, just write simple query, ORM will handle in internally.